apis/kubedb/v1alpha1/elasticsearch_types.go

```diff
@@ -153,6 +163,62 @@ type ElasticsearchNode struct {
 	MaxUnavailable *intstr.IntOrString `json:"maxUnavailable,omitempty" protobuf:"bytes,5,opt,name=maxUnavailable"`
 }
 
+type ElasticsearchTLSConfig struct {
```

Comment:
I wonder if we should make it more generic so that even if the we need more certs down the road, the YAML strucuture does not have to change?

```
type ElasticsearchTLSConfig struct {
	// IssuerRef is a reference to a Certificate Issuer.
	// +optional
	IssuerRef *core.TypedLocalObjectReference `json:"issuerRef,omitempty" protobuf:"bytes,1,opt,name=issuerRef"`

	// +optional
	Certificates []CertificateSpec `json:"transport,omitempty" protobuf:"bytes,2,opt,name=transport"`
}
```

Reply:
I added these types to client-go https://github.com/kmodules/client-go/pull/361 . Let's talk about it in the morning.